test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
That means that something is bringing in both the old MVC package and the new MVC package. Check the dependency graph of packages (you can try searching the project.lock.json file) to see what's bringing in Microsoft.AspNet.Mvc 6.x.